packages/playwright-core/src/server/injected/selectorGenerator.ts

```diff
@@ -551,7 +551,7 @@ function suitableTextAlternatives(text: string) {
 
   result = result.filter(r => r.text);
   if (!result.length)
-    result.push({ text: text.substring(0, 80), scoreBouns: 0 });
+    result.push({ text: trimWordBoundary(text, 80), scoreBouns: 0 });
```

Comment:
This was `substring()` on purpose, just in case line 548 did not yield anything.

Reply:
ok, reverted